test/scripts/scripts.test.js

```diff
@@ -6,29 +6,6 @@ import { waitForElement } from '../helpers/waitfor.js';
 document.head.innerHTML = await readFile({ path: './mocks/head.html' });
 document.body.innerHTML = await readFile({ path: './mocks/body.html' });
 
-const EXTERNAL_SCRIPTS = [
-  'https://www.adobe.com/marketingtech/main.standard.min.js',
-];
-
-// Prevents loading of EXTERNAL_SCRIPTS
-const observer = new MutationObserver((mutations) => {
-  mutations.forEach(({ addedNodes }) => {
-    addedNodes.forEach((node) => {
-      if (node.nodeType === 1 && node.tagName === 'SCRIPT') {
-        if (EXTERNAL_SCRIPTS.includes(node.src)) {
-          node.setAttribute('type', 'javascript/blocked');
-        }
-      }
-    });
-  });
-});
```

Comment:
This is very interesting; do you know if that worked, and it prevented loading the script? 

I'd assume the moment something is added to the page it would start loading - and the intersection observer would be too late to block loading it.

Reply:
I'm not sure, sorry. The marketingtech script was moved into the libs folder, breaking the test. Since it's no longer an external script, I removed that code, but I'm not familiar enough with the test previously to know if it worked before.

Reply:
To answer the question, it will in any case _download_ script but should stop _execution_